api/OpenAI.netstandard2.0.cs

```diff
@@ -4038,16 +4039,17 @@ public class ComputerOutput : IJsonModel<ComputerOutput>, IPersistableModel<Comp
         public override readonly string ToString();
     }
     public class FileSearchCallResponseItem : ResponseItem, IJsonModel<FileSearchCallResponseItem>, IPersistableModel<FileSearchCallResponseItem> {
+        public FileSearchCallResponseItem(IEnumerable<string> queries);
         public IList<string> Queries { get; }
-        public IList<FileSearchCallResult> Results { get; }
+        public IList<FileSearchCallResult> Results { get; set; }
```

Comment:
I believe we don't provide setter for collection properties? is it not the case here?

Reply:
Yes, this is correct. See my comment in PR description. This is a limitation in our codegen, but since this is experimental, I don't think it should be a blocker. 🙂